### ISSUE TITLE
First step not working

### ISSUE THREAD

**Issue by regortaz** [arjunskumar/Line-Follower--ROS#1]
Hello, the link you provided in the first step http://moorerobots.com/blog/ link seems to be broken. Do you have an alternative site that you can provide or do you have the original code there backed up? Thanks.

**Comment by p4lik4ri** [comment]
yeah could you please provide a link that is working i really need this tutorial

**Comment by regortaz** [comment]
> 

FYI, it's accessible from Wayback Machine
